CHANGELOG.md

```diff
@@ -169,6 +169,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* (x/tx/signing/textual) [#15730](https://github.com/cosmos/cosmos-sdk/pull/15730) make IntValueRenderer.Parse: gracefully handle "" + fuzz
```

Comment:
nit: tx is its own go mod, so the changelog should be there.

Reply:
Done, thanks @julienrbrt!